doc/Type/IO/Path.pod

```diff
@@ -211,7 +211,7 @@ http://en.wikipedia.org/wiki/File_system_permissions#Numeric_notation>
 specified as an integer (thought still subject to L<umask|
 http://en.wikipedia.org/wiki/Umask>).
 
-Throws an exception of type L<X::IO::Mkdir|/type/X::IO::Mkdir> if the
+Throws an exception of type L<X::IO::Mkdir> if the
```

Comment:
I did a mechanical search and replace that also caught this.  Dropping the link seems to work alright but I'm happy to back it out again.